src/components/home/whatWeDo.tsx

```diff
@@ -61,15 +78,92 @@ const StyledProcessItemExpand = styled.div<IStyledProcessItemExpand>`
     `}
 `;
 
-const StyledPill = styled.span`
+const marginLeftValues = [
+  "90px",
+  "0px",
+  "86px",
+  "0px",
+  "43px",
+  "11px",
+  "33px",
+  "0px",
+  "91px",
+  "0px",
+];
+const marginRightValues = [
+  "13px",
+  "56px",
+  "0px",
+  "91px",
+  "18px",
+  "62px",
+  "76px",
+  "61px",
+  "0px",
+  "0px",
+];
+
+const StyledPill = styled.span<{ index: number }>`
   background-color: ${colors.white};
   color: ${colors.black};
-  padding: 10px 30px;
-  margin: 0 16px;
+  padding: 10px 30px 12px 30px;
+  margin: 0px 16px;
   border-radius: 50px;
   font-weight: 500;
-  letter-spacing: 1px;
+  letter-spacing: 2px;
   display: inline-block;
+
+  @media all and (max-width: ${breakpoints.md}px) {
+    margin: 12px 16px;
+    text-align: center;
+    max-width: 350px;
+    font-size: ${getRemSize(dimensions.textSizes.normal.desktop)};
+    margin-left: ${(props) => marginLeftValues[props.index] || "0px"};
+    margin-right: ${(props) => marginRightValues[props.index] || "0px"};
```

Comment:
This is a bit more elegant of a solution. So basically using if statements can bulk our code up, so when we have so many in one go, there's usually a nicer more efficient way to do things. So it makes it easier for use to read and the code more efficient :)